cpu/cc26x0/periph/rfc.c

```diff
@@ -76,35 +102,36 @@ void rfc_beacon(void)
         printf("%.2x", ((uint8_t *) rop)[i]);
     printf("\n");
 
-    run_command_ptr(&rop->op);
-}
+    //run_command_ptr(&rop->op);
+}*/
 
 void rfc_prepare(void)
 {
+    printf("===> %s <===\n",__FUNCTION__);
+    /* RFC POWER DOMAIN CLOCK GATE */
     PRCM->RFCCLKG = 1;
 
     PRCM->CLKLOADCTL = CLKLOADCTL_LOAD;
     while (!(PRCM->CLKLOADCTL & CLKLOADCTL_LOADDONE)) ;
 
-    PRCM->PDCTL0RFC = 1;
-    PRCM->PDCTL1RFC = 1;
+    /* RFC POWER DOMAIN */
+    PRCM->PDCTL0RFC = PDCTL0_RFC_ON;;
```

Comment:
the macro for this would be `#define PDCTL0RFC_ON 1` because the register is PDCTL0RFC, not PDCTL0:RFC.
but because the register only has one bit it's probably not worth it. but then again, by now both of us have already...
you know ;-)